### PR TITLE
Find First Non-letter exercise #1

### PR DESCRIPTION
**Pull request by ines-radovanovic**
@jc15100 verified it works!
Now PUSH-IT!

### CODE REVIEW

File: c/findNonLetter.c [hunk 1]
```diff
@@ -15,9 +15,13 @@
  * * index of first non-letter if found
  * * -1 if not found
  */
-int findFirstNonLetter(char** arrayInput) {
+int findFirstNonLetter(char* arrayInput) {
 	// Task 1: Fill in the code here
-	
+    int n = sizeof(arrayInput) / sizeof(arrayInput[0]);
+    for (int index = 0; index < sizeof(n); index++){
```
Comment (jc15100) [inline c/findNonLetter.c]: why do you need the index to run until `sizeof(n)` and not just `n`?

File: c/findNonLetter.c [hunk 2]
```diff
@@ -15,9 +15,13 @@
...
+        if(!(('A' <= arrayInput[index]  && arrayInput[index] <= 'Z')  ||  ('a' <= arrayInput[index]  && arrayInput[index] <= 'z')))
```
Comment (jc15100) [inline c/findNonLetter.c]: nit: there is an extra space after `arrayInput[index]`

Comment (jc15100) [inline c/findNonLetter.c]: Can you add a comment before this if explaining in 2 lines how this works?

File: c/findNonLetter.c [hunk 3]
```diff
@@ -34,10 +38,11 @@ int main( int argc, char** argv) {
 	printf("Array Supplied: %s\n", argv[1]);
 
 	// find the first non-letter in the provided input
-	int indexOfFirstNonLetter = findFirstNonLetter(argv);
+	int indexOfFirstNonLetter = findFirstNonLetter(argv[1]);
 	
 	// Task 2: Print the index found and the actual non-letter character at that index
-	
+    printf("Index of first non-letter: %d\n", indexOfFirstNonLetter);
+    printf("First non-letter character: %c\n", argv[1][indexOfFirstNonLetter]);
```
Comment (jc15100) [inline c/findNonLetter.c]: 👍

File: c/findNonLetter.c (hunk 1 above)
Comment (ines-radovanovic) [inline c/findNonLetter.c]: You are right, forgot to replace sizeof(n) with just n

File: c/findNonLetter.c [hunk 4]
```diff
@@ -15,9 +15,15 @@
  * * index of first non-letter if found
  * * -1 if not found
  */
-int findFirstNonLetter(char** arrayInput) {
+int findFirstNonLetter(char* arrayInput) {
 	// Task 1: Fill in the code here
-	
+    int n = sizeof(arrayInput) / sizeof(arrayInput[0]);
+    // For-loop sweep through each character of the string comparing against A-Z and a-z
```
Comment (jc15100) [inline c/findNonLetter.c:21]: 👍

**Comment by jc15100** [comment]
You can merge. Also, give me write access to your repo.

**Pull request merged**